ops/internal/config/address.go

```diff
@@ -43,6 +45,30 @@ func (a ChecksummedAddress) String() string {
 	return common.Address(a).Hex()
 }
 
-func (a ChecksummedAddress) MarshalJSON() ([]byte, error) {
-	return []byte(fmt.Sprintf(`"%s"`, common.Address(a).Hex())), nil
+func (a *ChecksummedAddress) UnmarshalJSON(data []byte) error {
+	var dataStr string
+	if err := json.Unmarshal(data, &dataStr); err != nil {
+		return fmt.Errorf("failed to unmarshal ChecksummedAddress: %w", err)
+	}
+
+	if len(dataStr) != 42 {
+		return fmt.Errorf("invalid address length: %s", dataStr)
+	}
+
+	if !common.IsHexAddress(dataStr) {
+		return fmt.Errorf("invalid hex address: %s", dataStr)
+	}
+
+	// Convert to checksummed address
+	addr := common.HexToAddress(dataStr)
+	*a = ChecksummedAddress(addr)
+	return nil
+}
+
+func (a *ChecksummedAddress) MarshalJSON() ([]byte, error) {
+	if bytes.Equal(common.Address(*a).Bytes(), common.Address{}.Bytes()) {
```

Comment:
Why do you need a bytes.Equal here rather than just doing `common.Address(*a) == common.Address{}`?

Reply:
Good catch. We don't